### DISCUSSION TITLE
v0.9.0

### DISCUSSION THREAD

**Discussion by mikeu** [vue-leaflet/vue-leaflet#285] [Announcements]
### Added

- [Vitest](https://vitest.dev/), and a few initial tests.
- TypeScript type declarations in compiled output.

### Changed

- Converted codebase to TypeScript, finally closing [#1 Typescript support](https://github.com/vue-leaflet/vue-leaflet/issues/1).

<hr /><em>This discussion was created from the release <a href='https://github.com/vue-leaflet/vue-leaflet/releases/tag/v0.9.0'>v0.9.0</a>.</em>

**Comment by nosehair** [comment]
Really nice to see this project getting some traction again. Thanks for the hard work!
